Etabs_Adapter/Convert/Execute.cs

```diff
@@ -0,0 +1,233 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2024, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using BH.Engine.Adapter;
+using BH.oM.Adapters.ETABS;
+using BH.oM.Adapter;
+using BH.oM.Base;
+using BH.oM.Adapter.Commands;
+using BH.oM.Structure.Loads;
+
+#if Debug16 || Release16
+using ETABS2016;
+#elif Debug17 || Release17
+using ETABSv17;
+#else
+using CSiAPIv1;
+#endif
+
+
+namespace BH.Adapter.ETABS
+{
+#if Debug16 || Release16
+    public partial class ETABS2016Adapter
+#elif Debug17 || Release17
+    public partial class ETABS17Adapter
+#else
+    public partial class ETABSAdapter
+#endif
+    {
+        /***************************************************/
+        /**** IAdapter Interface                        ****/
+        /***************************************************/
+
+        public override Output<List<object>, bool> Execute(IExecuteCommand command, ActionConfig actionConfig = null)
+        {
+            var output = new Output<List<object>, bool>() { Item1 = null, Item2 = false };
+
+            output.Item2 = RunCommand(command as dynamic);
+
+            return output;
+        }
```

Comment:
Have we actually made any changes to this file? Doesn't look like it.

I think you've moved a file by accident.